cookies/cookies-impl/src/main/java/com/duckduckgo/cookies/impl/features/firstparty/FirstPartyCookiesModifierWorker.kt

```diff
@@ -72,18 +77,22 @@ class FirstPartyCookiesModifierWorkerScheduler @Inject constructor(
         .build()
 
     override fun onStop(owner: LifecycleOwner) {
-        if (isFeatureEnabled()) {
-            workManager.enqueueUniquePeriodicWork(FIRST_PARTY_COOKIES_EXPIRE_WORKER_TAG, ExistingPeriodicWorkPolicy.REPLACE, workerRequest)
-        } else {
-            workManager.cancelAllWorkByTag(FIRST_PARTY_COOKIES_EXPIRE_WORKER_TAG)
+        appCoroutineScope.launch(dispatchers.io()) {
```

Comment:
@CDRussell why not use the `owner`? like `owner.lifecycleScope.launch {`.

Reply:
If it were something we'd want to cancel (with the `lifecycleScope`) I would have used that but seemed more appropriate to use `appCoroutineScope` since we'd want scheduling the work to be non-cancellable I think

Reply:
Thought some more and decided it really shouldn't matter much either way; I've taken your suggestion to use `owner.lifecycleScope`